website/docs/getting-started/build-a-sample-app.mdx

```diff
@@ -80,7 +80,7 @@ that updates its value when clicked. Replace the contents of `src/main.rs` with
 :::note
 The call to `yew::Renderer::<App>::new().render()` inside the `main` function starts your application and mounts
 it to the page's `<body>` tag. If you would like to start your application with any dynamic
-properties, you can instead use `yew::Renderer::<App>::with_props(..).render()`.
```

Comment:
The original description is trying to say that `with_props()` can be used to set properties of `App`.

This should not be confused with `with_root()`, which can be used to set the root element where the renderer renders `App` into.